Tests/QuickTests/QuickTests/Fixtures/FunctionalTests_BehaviorTests_Behaviors.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  FunctionalTests_BehaviorTests_Behaviors.swift
+//  Quick
+//
+//  Created by Mostafa Amer on 19.04.17.
+//  Copyright © 2017 Brian Ivan Gesiak. All rights reserved.
+//
```

Comment:
You can remove this header and the ones below. Most of the files in this project don't include header comments.